src/test/java/org/swisspush/redisques/slowdown/EnqueueThrottleTest.java

```diff
@@ -0,0 +1,110 @@
+package org.swisspush.redisques.slowdown;
+
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Vertx;
+import io.vertx.core.VertxOptions;
+import io.vertx.core.eventbus.Message;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.RunTestOnContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.swisspush.redisques.RedisQues;
+import org.swisspush.redisques.util.QueueConfiguration;
+import org.swisspush.redisques.util.RedisquesAPI;
+import org.swisspush.redisques.util.RedisquesConfiguration;
+
+import java.util.Arrays;
+import java.util.HashSet;
+
+import static org.swisspush.redisques.util.RedisquesAPI.*;
+
+@RunWith(VertxUnitRunner.class)
+public class EnqueueThrottleTest {
+
+    private static final String REDISQUES_ADDRESS = "EnqueueThrottleTest-RedisQues";
+    private static final String PROCESSOR_ADDRESS = "EnqueueThrottleTest-Processor";
+    private static final String QUEUE_NAME = "EnqueueThrottleTest-q1";
+
+    @ClassRule
+    public static RunTestOnContext RULE = new RunTestOnContext(new VertxOptions().setBlockedThreadCheckInterval(10_000));
+
+    @Before
+    public void deployRedisQuesAndRegisterProcessor(TestContext testContext) {
```

Comment:
Really minor, the Q in deployRedisQuesAndRegisterProcessor should be lower case I think.

Reply:
The uppercase Q is used elsewhere too like the class `RedisQues`. I would keep it this way.

Reply:
But not in RedisquesAPI ?

Reply:
Apparently not :-)